src/sorter.rs

```diff
@@ -347,7 +347,11 @@ impl EntryBoundAlignedBuffer {
         let size = (size + entry_bound_size - 1) / entry_bound_size * entry_bound_size;
         let layout = Layout::from_size_align(size, align_of::<EntryBound>()).unwrap();
         let ptr = unsafe { alloc(layout) };
-        assert!(!ptr.is_null(), "the allocator is unable to allocate that much memory");
+        assert!(
+            !ptr.is_null(),
+            "the allocator is unable to allocate that much memory ({:#x} bytes requested)",
```

Comment:
```suggestion
            "the allocator is unable to allocate that much memory ({} bytes requested)",
```

I would prefer that we format that in decimal, easier to read for me :)

Reply:
Done exactly as suggested